app/components/Campus/AddCampus.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import store from '../../store';
+import {addCampus} from '../../reducers'
+import axios from 'axios';
+
+export default class AddCampus extends Component {
+  constructor(props) {
+    super(props);
+    this.state = store.getState();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    this.unsusbscribe = store.subscribe(() => {
+      this.setState(store.getState())
+    })
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    // console.log('inputCampusName: ', event.target.querySelector('#inputCampusName').value)
+    // console.log('inputCampusImage: ', event.target.querySelector('#inputCampusImage').value)
+
+    const name = event.target.querySelector('#inputCampusName').value;
+    const image = event.target.querySelector('#inputCampusImage').value;
+
+    axios.post('/api/campuses/addcampus', {name, image})
+    .then(res => res.data)
+    .then(campus => {
+      const addCampusAction = addCampus(campus)
+      store.dispatch(addCampusAction);
+    })
+    .catch(err => {
+      console.log(err)
+    })
+
+    event.target.querySelector('#inputCampusName').value = '';
+    event.target.querySelector('#inputCampusImage').value = '';
```

Comment:
I gave you this tip. But I came across something even better.

If the form inputs have `name` attributes the values can be even more easily accessed:

```html
<form onSubmit={handleFormSubmit}>
  <input name='inputA'/>
  <input name='inputB'/>
  <input name='inputC'/>
</form>
```

```js
function handleFormSubmit (event) {
  event.target.inputA.value
  event.target.inputC.value
  event.target.inputD.value
}
```